secrets/secrets.go

```diff
@@ -4,6 +4,11 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"io/fs"
+	"io/ioutil"
```

Comment:
`io/ioutil` is already deprecated, new code should use the replacements (usually from either `io` or `os`) instead.